db/migrate/20171114183148_profiles.rb

```diff
@@ -0,0 +1,16 @@
+Sequel.migration do
+
+  up do
```

Comment:
carto has a migration helper to avoid Postgres lockups: lib/carto/db/migration_helper.rb

Since these migrations were already applied though it may not make sense to modify these, but consider using the helper going forward.

Reply:
Good to know. I'll take a look in the future but in general the overhead of adding a column should be minimal

Reply:
The helper won't interfere with normal operations. It should be used always for safety!

No need in your installation though, just looking for consistency.